src/backend/distributed/master/master_modify_multiple_shards.c

```diff
@@ -266,81 +265,22 @@ static HTAB *
 OpenConnectionsToAllShardPlacements(List *shardIntervalList)
```

Comment:
How similar is this function to the one in multi_utility.c? Would it be possible to unify them?